frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.tsx

```diff
@@ -163,6 +164,8 @@ export function PublicOrEmbeddedDashboardView({
     [externalGetClickActionMode, navigateToNewCardFromDashboard],
   );
 
+  const isCompactHeader = !titled && !hasVisibleParameters && !dashboardHasTabs;
```

Comment:
Show a very compact header when there's no title, no parameters and no tabs.

![CleanShot 2568-04-11 at 06 15 51@2x](https://github.com/user-attachments/assets/201e41cc-af23-49d8-9f49-da9f7807ef82)